### ISSUE TITLE
Create Lambda Handler

### ISSUE THREAD

**Issue by apolaskey** [Ukraine-Relief-Efforts/scraper-translator#3]
A lambda event handler needs to be created to process the event invocation from AWS.

Refer to: https://github.com/awsdocs/aws-lambda-developer-guide/blob/main/sample-apps/blank-nodejs/function/index.js as an example.

**Comment by WillDHB** [comment]
Don't we have one already?

**Comment by apolaskey** [comment]
No, we have lots of helper functions; we need an actual handler...

```
exports.handler = async function(event, context) {
    // Do the work
}
```

From in here we then call whatever logic we need for the lambda itself

**Comment by WillDHB** [comment]
Oh, sorry, I thought this was the scraper repo...nevermind :/

**Comment by apolaskey** [comment]
See https://github.com/Ukraine-Relief-Efforts/scraper-translator/pull/5 includes a basic scaffold for the handler and some organization for any future work needed.

**Comment by Jachdich** [comment]
Pretty sure this was fixed by #5, closing
